controllers/parameters.go

```diff
@@ -34,14 +34,15 @@ const (
 )
 
 // filterPrefixedParameters removes all the reserved keys from the
-// replicationclass which are matching the prefix.
+// replication class which are matching the prefix.
 func filterPrefixedParameters(prefix string, param map[string]string) map[string]string {
 	newParam := map[string]string{}
 	for k, v := range param {
 		if !strings.HasPrefix(k, prefix) {
 			newParam[k] = v
 		}
 	}
+
```

Comment:
haven't seen this issue in CI. can you please enable/update the linter version where you are seeing this one?

Reply:
version `1.39` was showing these errors @Madhu-1

Reply:
please update the CI to this version to verify changes and to avoid same mistakes in future

Reply:
@Madhu-1 done..thanks.